### DISCUSSION TITLE
Help with UniDec UPP for DAR Calculation and Table Setup

### DISCUSSION THREAD

**Discussion by Novotle7** [michaelmarty/UniDec#100] [Q&A] [answered]
Hi everyone,

I’m trying to use UniDec’s Process Pipeline (UPP) to calculate the drug-to-antibody ratio (DAR), but I’m having trouble getting DAR results. So far, I’m getting deconvolution data, but no DAR values, and I’m not sure what to adjust to make it work.

I’m using these parameters:

    Sample name
    Data Directory
    Config Peak Thres
    Config High Mass
    Config Low Mass
    Config High m/z
    Config Low m/z
    Config Sample Mass Every
    Drug Mass
    Min Drug
    Max Drug
    Tolerance (Da)
    Protein Mass

If anyone has tips on how to set up the table or change these settings to calculate DAR, I’d be grateful. Test data or examples would also be really helpful.

Thanks a lot for your help!
Lenka

**Comment by michaelmarty** [comment, marked as answer]
Hi Lenka,
Sorry for the slow response. I was out of town for a week. 

You are really close. Max Drug should be Max Drugs. Similarly, Min Drug should be Min Drugs. I think all the others were right. This was my fault; I had it typed wrong in the help file. Sorry for that! I've fixed it for the next version. 

Thanks for your help debugging!
MTM

**Reply by Novotle7** [reply]
Hi Marty,

Thank you very much for your response and for providing such an easy solution. Unfortunately, I’m facing a different issue—I can’t start UniDec. My antivirus software is blocking it, and the IT department informed me that it’s being flagged as a Trojan Horse. Would it be possible to check the current version of UniDec and address this problem?

Thank you very much for your help.

Lenka

**Reply by michaelmarty** [reply]
Hi Lenka,
That's a tough one for me to solve, but I have a few ideas. 

First, can you give any information on specific files or processes that are getting flagged? I can try to see if it's a general or specific problem. 

Second, if it's helpful, I can send a checksum for the file. Some IT departments have asked for this in the past to help verify the file. 

Third, you can try this version to see if it works better: https://www.dropbox.com/scl/fi/jz7gb7bu7ijike22nasys/UniDec_Windows_240930.zip?rlkey=tip2nqtbewsbi5zog68qpynmx&dl=0. Might be something about how it's getting downloaded. 

Finally, if all else fails, could you try running it from Python? If you are able to install Python on your computer, you can do pip install unidec to get it installed and python -m unidec.Launcher to launch it. 

Let me know how those go, and we'll see if we can work through it. Thanks,
MTM

**Reply by Novotle7** [reply]
Hi Marty,

Thank you for your suggestions and assistance. I used the link you provided, and UniDec is now functioning correctly, including the DAR calculation. 

Thanks again for your help and the great support.

Lenka

**Reply by michaelmarty** [reply]
Awesome! Let me know if you have issues in the future. Thanks!
